stable/pomerium/templates/_helpers.tpl

```diff
@@ -15,6 +15,11 @@
 {{- default (printf "%s-authenticate" .Chart.Name) .Values.authenticate.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*Expand the name of the authenticate-service.*/}}
+{{- define "pomerium.authenticate.internalName" -}}
+{{- default (printf "%s-authenticate-internal" .Chart.Name) .Values.authenticate.internalNameOverride | trunc 63 | trimSuffix "-" -}}
```

Comment:
maybe we should add also `internalNameOverride` and `internalFullnameOverride` as commented in values.yaml?

Reply:
👍